From: =?utf-8?q?Guido_G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sun, 23 Jul 2017 13:56:20 +0200
Subject: debian: switch to archive URLs for stretch

Now that the first point release is there switch to archive URLs since
these remain stable (at least as long as this is the current stable
release). The current URLs are now longer valid since the patchs contain
a 9.0.1 in the filename now.
---
 data/os/debian.org/debian-9.xml.in | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/data/os/debian.org/debian-9.xml.in b/data/os/debian.org/debian-9.xml.in
index 80cc048..759422b 100644
--- a/data/os/debian.org/debian-9.xml.in
+++ b/data/os/debian.org/debian-9.xml.in
@@ -49,7 +49,7 @@
     <!-- Network installer -->
     <media arch="i686">
       <variant id="universal-netinst"/>
-      <url>http://cdimage.debian.org/cdimage/release/current/i386/iso-cd/debian-9.0.0-i386-netinst.iso</url>
+      <url>http://cdimage.debian.org/cdimage/archive/9.0.0/i386/iso-cd/debian-9.0.0-i386-netinst.iso</url>
       <iso>
         <volume-id>Debian 9.(\d)+.(\d)+ i386 n</volume-id>
       </iso>
@@ -58,7 +58,7 @@
     </media>
     <media arch="x86_64">
       <variant id="universal-netinst"/>
-      <url>http://cdimage.debian.org/cdimage/release/current/amd64/iso-cd/debian-9.0.0-amd64-netinst.iso</url>
+      <url>http://cdimage.debian.org/cdimage/archive/9.0.0/amd64/iso-cd/debian-9.0.0-amd64-netinst.iso</url>
       <iso>
 	<volume-id>Debian 9.(\d)+.(\d)+ amd64 n</volume-id>
       </iso>
@@ -67,7 +67,7 @@
     </media>
     <media arch="aarch64">
       <variant id="universal-netinst"/>
-      <url>http://cdimage.debian.org/cdimage/release/current/arm64/iso-cd/debian-9.0.0-arm64-netinst.iso</url>
+      <url>http://cdimage.debian.org/cdimage/archive/9.0.0/arm64/iso-cd/debian-9.0.0-arm64-netinst.iso</url>
       <iso>
 	<volume-id>Debian 9.(\d)+.(\d)+ arm64 n</volume-id>
       </iso>
@@ -76,7 +76,7 @@
     </media>
     <media arch="armv7l">
       <variant id="universal-netinst"/>
-      <url>http://cdimage.debian.org/cdimage/release/current/armhf/iso-cd/debian-9.0.0-armhf-netinst.iso</url>
+      <url>http://cdimage.debian.org/cdimage/archive/9.0.0/armhf/iso-cd/debian-9.0.0-armhf-netinst.iso</url>
       <iso>
 	<volume-id>Debian 9.(\d)+.(\d)+ armhf n</volume-id>
       </iso>
@@ -87,7 +87,7 @@
     <!-- DVD -->
     <media arch="i686">
       <variant id="universal"/>
-      <url>http://cdimage.debian.org/cdimage/release/9.0.0/i386/iso-dvd/debian-9.0.0-i386-DVD-1.iso</url>
+      <url>http://cdimage.debian.org/cdimage/archive/9.0.0/i386/iso-dvd/debian-9.0.0-i386-DVD-1.iso</url>
       <iso>
         <volume-id>Debian 9.(\d)+.(\d)+ i386 1</volume-id>
       </iso>
@@ -96,7 +96,7 @@
     </media>
     <media arch="x86_64">
       <variant id="universal"/>
-      <url>http://cdimage.debian.org/cdimage/release/9.0.0/amd64/iso-dvd/debian-9.0.0-amd64-DVD-1.iso</url>
+      <url>http://cdimage.debian.org/cdimage/archive/9.0.0/amd64/iso-dvd/debian-9.0.0-amd64-DVD-1.iso</url>
       <iso>
 	<volume-id>Debian 9.(\d)+.(\d)+ amd64 1</volume-id>
       </iso>
@@ -105,7 +105,7 @@
     </media>
     <media arch="aarch64">
       <variant id="universal"/>
-      <url>http://cdimage.debian.org/cdimage/release/9.0.0/arm64/iso-dvd/debian-9.0.0-arm64-DVD-1.iso</url>
+      <url>http://cdimage.debian.org/cdimage/archive/9.0.0/arm64/iso-dvd/debian-9.0.0-arm64-DVD-1.iso</url>
       <iso>
 	<volume-id>Debian 9.(\d)+.(\d)+ arm64 1</volume-id>
       </iso>
@@ -114,7 +114,7 @@
     </media>
     <media arch="armv7l">
       <variant id="universal"/>
-      <url>http://cdimage.debian.org/cdimage/release/9.0.0/armhf/iso-dvd/debian-9.0.0-armhf-DVD-1.iso</url>
+      <url>http://cdimage.debian.org/cdimage/archive/9.0.0/armhf/iso-dvd/debian-9.0.0-armhf-DVD-1.iso</url>
       <iso>
 	<volume-id>Debian 9.(\d)+.(\d)+ armhf 1</volume-id>
       </iso>
